hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/HLESurveyTransformationPipelineBuilder.scala

```diff
@@ -6,19 +6,25 @@ import com.spotify.scio.values.SCollection
 import org.broadinstitute.monster.common.{PipelineBuilder, StorageIO}
 import org.broadinstitute.monster.common.msg._
 import org.broadinstitute.monster.dogaging.jadeschema.table.{HlesDog, HlesOwner}
+import org.slf4j.LoggerFactory
 import upack.Msg
 
 object HLESurveyTransformationPipelineBuilder extends PipelineBuilder[Args] {
 
   implicit val msgCoder: Coder[Msg] = Coder.beam(new UpackMsgCoder)
 
   case class RawRecord(id: Long, fields: Map[String, Array[String]]) {
+    val logger = LoggerFactory.getLogger(getClass)
```

Comment:
Is this used anywhere?

Reply:
Ope, not anymore! Good catch. Just removed it.